.vuepress/public/postman/meilisearch-collection.json

```
@@ -1192,6 +1207,81 @@
 								}
 							},
 							"response": []
+						},
+						{
+							"name": "Get typo-tolerance",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{url}}/indexes/{{indexUID}}/settings/typo-tolerance",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"indexes",
+										"{{indexUID}}",
+										"settings",
+										"typo-tolerance"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Update typo-tolerance",
+							"protocolProfileBehavior": {
+								"disabledSystemHeaders": {
+									"content-type": true
+								}
+							},
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"minWordLengthForTypo\": {\n        \"oneTypo\": 2,\n        \"twoTypos\": 11\n    },\n    \"disableOnWords\": [\"Prince\"],\n    \"disableOnAttributes\": [\"genre\"]\n\n}"
```

```suggestion
									"raw": "{\n    \"minWordSizeForTypos\": {\n        \"oneTypo\": 2,\n        \"twoTypos\": 11\n    },\n    \"disableOnWords\": [\"Prince\"],\n    \"disableOnAttributes\": [\"genre\"]\n\n}"
```

Shouldn't this be `minWordSizeForTypos`?

---

Sorry, I did it using v0.27.orc1, I should have waited for rc2 before posting this PR. Changing this!